packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -445,4 +468,39 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
             }
         );
     }
+
+    protected async shouldCreateBeforeOpen(uri: URI): Promise<boolean> {
+        // Do not try to create a document which scheme is not a file
+        if (uri.scheme !== 'file') {
+            return false;
```

Comment:
I think we should do https://github.com/eclipse-theia/theia/issues/7127 after that we will have the proper file system API which respect all kind of schemes.